packages/sync-actions/src/products.js

```diff
@@ -38,6 +38,11 @@ function createProductMapActions (mapActionGroup) {
     allActions.push(mapActionGroup('variants', () =>
       productActions.actionsMapVariants(diff, oldObj, newObj)))
 
+    const changeMasterVariantActions =
+      productActions.actionsMapMasterVariant(oldObj, newObj)
+    if (changeMasterVariantActions.length > 0)
```

Comment:
I think we don't need the condition again, if it's empty array, it's fine for us to push it.

Reply:
Ah ok. Thought better to be safe than sorry. Here you go f8a6616